packages/react/src/lib/felaRenderer.tsx

```diff
@@ -9,6 +9,7 @@ import { Renderer } from '../themes/types'
 
 const createRendererConfig = (options: any = {}) => ({
   devMode: process.env.NODE_ENV !== 'production',
+  selectorPrefix: '_', // https://github.com/rofrischmann/fela/issues/192
```

Comment:
Move to the same place where we set it for RTL (line 29)

Reply:
@miroslavstastny mm.. `selectorPrefix` is not overridden by RTL, or are you suggesting that just so we have it consistent and in one place?

Reply:
nvm we're going with #951